### PR TITLE
Changed README to help connect to local host

### PR DESCRIPTION
**Pull request by jeramief**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
+
+3. In the config.env file, make sure there is a number after PORT for local hosting i.e. **PORT = 7777**
+
 4. Navigate to your MongoDB Atlas database connection and find your Database Connection String
+
 5. Paste after **DB_STRING =** (Note: ensure you update the `password` to your database password)
+
 6. Save the config.env file.
+
 7. Execute ```npm start``` to start the server. If you prefer, you can also use ```nodemon start``` instead to automatically reflect any changes.
```
Comment (IntelliJinceTech) [inline README.md:17]: This seems redundant --> `If you prefer, you can also use ```nodemon start``` instead to automatically reflect any changes.`

**Comment by IntelliJinceTech** [comment]
I just made a comment b/c I think there's some redundancy in the readme. If you feel that it's still needed, I'm okay with the merge.

File: README.md (hunk 1 above)
Comment (jeramief) [reply README.md]: I deleted that sentence.

**Comment by ImanKahlila** [comment]
I'd like to outline my own difficulties when I was trying to setup my MongoDB account, npm install, and localhost in case it may help someone else. We could include it in the README possibly.

I had to Create A Database and Create A Collection in order to get a password generated for my MongoDB Atlas account.
I'm not sure if anyone else had to do this or not.

I also tried the "MongoDB for VS code" option first because I didn't know I had to click the "Drivers" option instead.
It seems as though my localhost didn't start working until I connected "Drivers" but don't quote me on that.
mongodb

**Review by IntelliJinceTech — APPROVED** [review]
Looks good to me.

**Comment by IntelliJinceTech** [comment]
> I'd like to outline my own difficulties when I was trying to setup my MongoDB account, npm install, and localhost in case it may help someone else. We could include it in the README possibly.
> 
> I had to Create A Database and Create A Collection in order to get a password generated for my MongoDB Atlas account. I'm not sure if anyone else had to do this or not.
> 
> I also tried the "MongoDB for VS code" option first because I didn't know I had to click the "Drivers" option instead. It seems as though my localhost didn't start working until I connected "Drivers" but don't quote me on that. mongodb

We can certainly add a troubleshooting section to the `readme` but I think for the sake of this PR, it looks good to me. @ImanKahlila if you are still having issues, I could try to help troubleshoot and see where that leads us.

**Review by Cheesely — APPROVED** [review]

**Comment by Cheesely** [comment]
Looks good to me too. I will go ahead and merge.

**Pull request merged**